sound/soc/sof/intel/hda-dai.c

```diff
@@ -323,8 +322,8 @@ static int hda_link_pcm_trigger(struct snd_pcm_substream *substream,
 		 * clear link DMA channel. It will be assigned when
 		 * hw_params is set up again after resume.
 		 */
-		ret = hda_link_config_ipc(hda_stream, dai->name,
-					  DMA_CHAN_INVALID, substream->stream);
+		ret = hda_link_config_ipc(hda_stream, DMA_CHAN_INVALID,
+					  substream->stream, dai);
```

Comment:
probably best to keep the arguments in the same order for reviewers, i.e just a dai->name to dai change